From 92d0d2677c1825582001bfd25b95cb3a5594e1f2 Mon Sep 17 00:00:00 2001
From: Udayan Banerji <udayan.banerji@intel.com>
Date: Tue, 7 May 2013 11:28:42 -0700
Subject: Dalvik: Fix build problem with VTUNE_DALVIK flag off

BZ: 106435

There is a compilation error when VTUNE_DALVIK flag is off, affecting user
builds.

Category: device-enablement
Domain: AOSP-Dalvik-Compiler-CG
Origin: internal
Upstream-Candidate: no, needs rework

Change-Id: Ia8d87fb523158d18393b846ca43d9299e3206960
Orig-MCG-Change-Id: I3305bc3c701b76cb4279197bdabc403b8ae7f50f
Signed-off-by: Udayan Banerji <udayan.banerji@intel.com>
Signed-off-by: Qiming Shi <qiming.shi@intel.com>
Signed-off-by: Serguei Katkov <serguei.i.katkov@intel.com>
---
 vm/compiler/codegen/x86/CodegenInterface.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vm/compiler/codegen/x86/CodegenInterface.cpp b/vm/compiler/codegen/x86/CodegenInterface.cpp
index dfc8052..50c9dce 100644
--- a/vm/compiler/codegen/x86/CodegenInterface.cpp
+++ b/vm/compiler/codegen/x86/CodegenInterface.cpp
@@ -50,7 +50,7 @@ static void updateCodeCache(int &dst, int src)
 /* Send updated code cache content to VTune */
 static void SendUpdateToVTune(void * address, unsigned size, unsigned method_id = 0)
 {
-    (void) address; (void) size; (void) unsigned method_id;
+    (void) address; (void) size; (void) method_id;
 }
 
 #else
-- 
1.7.4.1

